frontend/src/css/modules/base.scss

```diff
@@ -1033,3 +1039,4 @@ h4.challenge-step-title {
         position: relative;
     }
 }
+
```

Comment:
@Alabhya268 can you remove extra newline. Rest of it LGTM

Reply:
ok

Reply:
@Ram81 ,I  have removed this line now.Let me know if anything else needs to be changed.